RetailCoder.VBE/Refactorings/ExtractMethod/ExtractMethodModel.cs

```diff
@@ -122,11 +78,19 @@ private void Setup()
 
             if (string.IsNullOrWhiteSpace(NewMethodName))
             {
-                NewMethodName = "NewMethod";
+                NewMethodName = RubberduckUI.ExtractMethod_DefaultNewMethodName;
             }
 
             SelectedCode = string.Join(Environment.NewLine, SelectedContexts.Select(c => c.GetText()));
 
+            var rawCode = string.Join(Environment.NewLine,
+                CodeModule.GetLines(1, CodeModule.CountOfLines));
+            var regex = new Regex(@"^(\s?)+#", RegexOptions.Multiline);
+            if (regex.Matches(rawCode).Count > 0)
+            {
+                ModuleContainsCompilationDirectives = true;
+            }
```

Comment:
can be `ModuleContainsCompilationDirectives = regex.Matches(rawCode).Count > 0` 

Also consider extracting this into an extension method on CodeModules to encapsulate the access of rawCode